### DISCUSSION TITLE
[Linux] osu! becomes windowed on alt+tab

### DISCUSSION THREAD

**Discussion by taep96** [ppy/osu#16345] [Q&A] [answered]
When changing to another tag and back in dwm osu! becomes windowed. To change it back I have to press f11 3 times. Can you make osu! force current setting like for example minecraft does? You can reproduce the issue without any configuration using dwm: https://dwm.suckless.org/.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/12360
